<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="TitleLabel" name="TitleLabel_2">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="HyperlinkButton" name="HyperlinkButton">
       <property name="text">
        <string>Bug Report</string>
       </property>
       <property name="url">
        <url>
         <string>https://github.com/deeffest/DEEF-Lite-Image-Viewer/issues/new/choose</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="HorizontalSeparator" name="HorizontalSeparator"/>
   </item>
   <item>
    <widget class="ScrollArea" name="ScrollArea">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>546</width>
        <height>707</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="rightMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="SubtitleLabel" name="SubtitleLabel_5">
              <property name="text">
               <string>Main</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="BodyLabel" name="BodyLabel_7">
               <property name="text">
                <string>Save last resolution of window</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="SwitchButton" name="SwitchButton_3">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_6">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="BodyLabel" name="BodyLabel_19">
               <property name="text">
                <string>App language</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="ComboBox" name="ComboBox">
               <property name="text">
                <string>en_EN</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="SubtitleLabel" name="SubtitleLabel_6">
              <property name="text">
               <string>Personalization</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="BodyLabel" name="BodyLabel_8">
               <property name="text">
                <string>App theme</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="RadioButton" name="RadioButton">
               <property name="text">
                <string>Dark</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="RadioButton" name="RadioButton_2">
               <property name="text">
                <string>Light</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="BodyLabel" name="BodyLabel_14">
               <property name="text">
                <string>Scene theme</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="RadioButton" name="RadioButton_3">
               <property name="text">
                <string>Dark</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="RadioButton" name="RadioButton_4">
               <property name="text">
                <string>Light</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="BodyLabel" name="BodyLabel_9">
               <property name="text">
                <string>App color theme</string>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="ComboBox" name="ComboBox_2">
               <property name="text">
                <string>Default</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item alignment="Qt::AlignLeft">
             <widget class="SubtitleLabel" name="SubtitleLabel_7">
              <property name="text">
               <string>Updates</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="BodyLabel" name="BodyLabel_10">
               <property name="text">
                <string>Search for updates at startup</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="SwitchButton" name="SwitchButton_4">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_8">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="BodyLabel" name="BodyLabel_11">
               <property name="text">
                <string>Current app version</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
               <property name="text">
                <string>0.1</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="PushButton" name="PushButton">
               <property name="text">
                <string>Check updates</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="SubtitleLabel" name="SubtitleLabel_8">
              <property name="text">
               <string>About</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_11">
            <layout class="QHBoxLayout" name="horizontalLayout_23">
             <item>
              <widget class="BodyLabel" name="BodyLabel_13">
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>DLIVIewer is a common image viewer for Windows, trying to stick to the principle:
- Lightweight,
- Fast,
- Open.</string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_9">
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="BodyLabel" name="BodyLabel_12">
               <property name="text">
                <string>Created with ♥ by deeffest, 2023-2024.</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="HyperlinkButton" name="HyperlinkButton_4">
               <property name="toolTipDuration">
                <number>-1</number>
               </property>
               <property name="text">
                <string>Github</string>
               </property>
               <property name="url">
                <url>
                 <string>https://github.com/deeffest</string>
                </url>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="HyperlinkButton" name="HyperlinkButton_5">
               <property name="text">
                <string>Web-site</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="url">
                <url>
                 <string>https://deeffest.pythonanywhere.com</string>
                </url>
               </property>
               <property name="hasIcon" stdset="0">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="HyperlinkButton" name="HyperlinkButton_6">
               <property name="text">
                <string>Donation</string>
               </property>
               <property name="url">
                <url>
                 <string>https://donationalerts.com/r/deeffest</string>
                </url>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HyperlinkButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
